docs/misc/making-docs-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to contribute to DataHub docs?
+date: 2017-12-21
+authors: ['Mikanebu']
+---
+
+
+### How to contribute to DataHub docs?
+
+You are always welcome to open PR for DataHub docs.
+Since you know how markdown works, there are only couple steps you need to know:
+
+* Template - template for home docs page is located in `views/docs_home.html`, the rest docs use `views/docs.html`
+* css - our default css class is `docs` which is located `/public/sass/_doc.scss`. We use preprocessors like [*Sass*][sass]
+* [Front Matter][frontmatter] - supports YAML format where you can define variables like *title, date, author* etc
+
+```
+---
+title: Getting data tutorial
+date: 2017-11-01
+---
+```
+
+**Note:** *file name should be lowercased without any white spaces*.
+
+### How to write blogs on DataHub?
```

Comment:
Why do we need this section? Title tells me only about contributing to the docs.